.github/workflows/add-labels.yml

```
@@ -0,0 +1,54 @@
+name: add labels for some cases
+
+on: pull_request
+
+jobs:
+  reviewapp:
+    runs-on: ubuntu-latest  # windows-latest | macos-latest
+    name: grant 'reviewapps' label if there are any changes in PR's source code.
+    if: ${{ ! startsWith(github.head_ref, 'renovate/') }}
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Get changed files
+        id: check-paths-ignore
+        uses: tj-actions/changed-files@v42
+        with:
+          files_ignore: |
+            .github/**/*.yml
+            **.md
+
+      # to trigger other Actions caused by adding reviewapp Label
+      - name: Generate token
+        if: steps.check-paths-ignore.outputs.any_changed == 'true'
+        id: generate_token
+        uses: tibdex/github-app-token@v2
+        with:
+          app_id: ${{ secrets.APP_ID }}
+          private_key: ${{ secrets.PRIVATE_KEY }}
+
+      - name: Labeling 'reviewapps' to PR
+        if: steps.check-paths-ignore.outputs.any_changed == 'true'
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{ steps.generate_token.outputs.token }}
+          result-encoding: string
+          script: |
+            const targetLabel = 'reviewapps';
+            issue = await github.rest.issues.get({
```

```suggestion
            const issue = await github.rest.issues.get({
```

nit: 小さなscriptだから気にしすぎなのですが、JavaScriptでconstもletもついていないとcode smellがするので、念の為

---

actions/labeler 利用に伴い指摘箇所が workflow からなくなったため、 resolve します。